integration_tests/tests/productInfo.cy.ts

```diff
@@ -0,0 +1,13 @@
+context('Product info', () => {
+  context('Info provided', () => {
+    beforeEach(() => {
+      cy.task('reset')
+      cy.task('stubAuthPing')
+      cy.task('stubTokenVerificationPing')
+    })
+
+    it('Product ID is visible', () => {
```

Comment:
I guess this from the test in the template we copied from?

Reply:
corrected!